.github/publish.yaml

```diff
@@ -0,0 +1,44 @@
+name: Build and publish the Blossom extension
+
+# GH release doc https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
+on:
+  release:
+    types: [published]
+    tags-ignore:
+      - blossom-js-lib*
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - name: Checkout
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache node modules
+        id: cache-npm-ext
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install dependencies
+        run: npm ci
+        if: ${{ steps.cache-npm-ext.outputs.cache-hit == 'true' }}
+
+      - name: Build
+        run: npm run build
+
+      # TODO Publish the extension
```

Comment:
I guess you do that now, don't you?

Reply:
For now, we should skip this step. There is no sense to publish the extension until it's finished.

Reply:
create an issue for that